adapter/src/dbt/adapters/fal/wrappers.py

```diff
@@ -20,6 +20,14 @@ def __init__(self, db_creds: Credentials):
 
     @property
     def type(self):
+        import inspect
+
+        target = filter(lambda s: s.function == "to_target_dict", inspect.stack())
+        db_mat = filter(lambda s: s.function == "db_materialization", inspect.stack())
+        if any(target) or any(db_mat):
```

Comment:
Just a small style nit:
```suggestion
        materializer_funcs = {"to_target_dict", "db_materialization"}
        if any(frame.function in materializer_funcs for frame in inspect.stack()):
 ```